### PR TITLE
Testing PR review process

### PR DESCRIPTION
**Pull request by gpehlev**

### CODE REVIEW

File: dummy.txt [hunk 1]
```diff
@@ -1 +1 @@
-Change on branch single-workflow-test-branch1
\ No newline at end of file
+Testing pull request review process
```
Comment (georgi-pehlev-epam) [inline dummy.txt]: Add review comment.

**Review by georgi-pehlev-epam — COMMENTED** [review]
Left a comment.

File: dummy.txt (hunk 1 above)
Comment (gpehlev) [reply dummy.txt]: Acknowledged comment, resolving.

**Review by georgi-pehlev-epam — APPROVED** [review]
PR approved

**Pull request merged**